instrumentation/src/main/java/io/opentelemetry/android/OtelAndroidConfig.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android;
+
+import io.opentelemetry.android.instrumentation.network.CurrentNetworkProvider;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.common.AttributesBuilder;
+
+/**
+ * Configuration object for OpenTelemetry Android. The configuration items in this class will be
+ * used in the OpenTelemetryRumBuilder to wire up and enable/disable various mobile instrumentation
+ * components.
+ */
+public class OtelAndroidConfig {
+
+    private AttributesBuilder globalAttributes = Attributes.builder();
+    private boolean includeNetworkAttributes = true;
+    private boolean generateSdkInitializationEvents = true;
+    private boolean includeScreenAttributes = true;
+
+    /**
+     * Configures the set of global attributes to emit with every span and event. Any existing
+     * configured attributes will be dropped. Default = none.
+     */
+    public OtelAndroidConfig setGlobalAttributes(Attributes attributes) {
```

Comment:
Not necessarily for this PR, although I'm thinking that some of the use cases for global attrs might require those attrs to be dynamic, for example the session id one, so I'm thinking that it might be better to provide a supplier of attrs rather than the attrs right away.

Reply:
Let's tackle that as a separate effort. As it stands right now, the session id isn't lumped in with the other global attributes, it's a separate concern. Global attributes are currently changeable through the `GlobalAttributesSpanAppender.update()` method, but I'm not married to that. There is room for improvement for sure.

Reply:
Sounds good, yeah it doesn't have to be for this PR though I took the opportunity to mention it. The session id is one example, connectivity info attrs would fit in here as well, maybe also user ids that would only become available after a login and would be removed/replaced after a logout, etc. The idea of having to manually call an update function to do so seems a bit easy to forget about, also people would have to spread telemetry-specific code across their project's classes in order to keep attrs updated with changes in their app's state, which might not be a huge problem depending on the app's architecture, but in general it seems handy/cleaner to have a single place to make these decisions in a reactive manner.

Reply:
I've created this issue to follow-up there: https://github.com/elastic/apm-agent-android/issues/216